test/e2e/e2e_suite_test.go

```diff
@@ -50,7 +51,7 @@ const (
 func TestE2E(t *testing.T) {
 	RegisterFailHandler(Fail)
 	// adding JUnitReporter for arc conformance test. If JUNIT_OUTPUT_FILEPATH is empty, it will not produce xml output.
-	junitReporter := reporters.NewJUnitReporter(os.Getenv("JUNIT_OUTPUT_FILEPATH"))
+	junitReporter := reporters.NewJUnitReporter(filepath.Join(os.Getenv("JUNIT_OUTPUT_FILEPATH"), "junit_report.xml"))
```

Comment:
The var we export for confomace test is - 
`export JUNIT_OUTPUT_FILEPATH=/tmp/results/junit.xml`

This will break while running conformance test.

Reply:
Where is that being set? 

I see this failing in the CI, so any recommendation on how we resolve this without breaking the conformance test?

Reply:
We set it while running conformance test - https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/arc/conformance/plugin/arc_conformance.sh#L48

The easiest way I can think right now is to check if `JUNIT_OUTPUT_FILEPATH` is empty or not and set above changes accordingly.

Reply:
Let's go with your approach. Let's update script with setting only dir path. Filename `junit.xml` we can set it in the code. This way it will be minimal changes without extra if checks etc.